lib/rest/RequestHandler.js

```diff
@@ -257,12 +257,14 @@ class RequestHandler {
                         }
 
                         if(resp.statusCode !== 429) {
-                            this._client.emit("debug", `${body && body.content} ${now} ${route} ${resp.statusCode}: ${latency}ms (${this.latencyRef.latency}ms avg) | ${this.ratelimits[route].remaining}/${this.ratelimits[route].limit} left | Reset ${this.ratelimits[route].reset} (${this.ratelimits[route].reset - now}ms left)`);
+                            const content = typeof body === "object" && body.content ? `${body.content} ` : "";
```

Comment:
Just check for `body`, not `body.content` (both ternary and actual event

Reply:
Gotcha! 👍